packages/website/scripts/generate-sitemap.ts

```diff
@@ -0,0 +1,50 @@
+import dotenv from 'dotenv';
+import { writeFileSync } from 'fs';
+import prettier from 'prettier';
+import { getMdxPaths } from '../utils/content';
+
+dotenv.config();
+
+async function generate() {
+  // todo: replace with f36.contentful.com when we release v4
+  const baseUrl = 'https://v4-forma-36.vercel.app';
+
+  const staticSlugs = [''];
```

Comment:
I guess this is for other pages that are not mdx based. And we will have to remember to include them here, right?

Reply:
yes, I'm going to add a comment

Reply:
added a comment